src/auro-calendar-cell.js

```diff
@@ -0,0 +1,151 @@
+import { LitElement, html } from 'lit';
+import { format, getTime, startOfDay } from 'date-fns';
+import { enUS } from 'date-fns/esm/locale';
+
+import styleCss from "./style-auro-calendar-cell-css";
+
+/* eslint-disable no-magic-numbers, no-underscore-dangle, max-params, no-void, init-declarations, no-extra-parens, arrow-parens */
+
+export class AuroCalendarCell extends LitElement {
+  constructor() {
+    super();
+
+    this.day = null;
+    this.selected = false;
+    this.hovered = false;
+    this.dateTo = null;
+    this.dateFrom = null;
+    this.month = null;
+    this.min = null;
+    this.max = null;
+    this.disabled = false;
+    this.disabledDays = [];
+    this.hoveredDate = null;
+    this.isCurrentDate = false;
+    this._locale = null;
+  }
+
+  // This function is to define props used within the scope of this component
+  // Be sure to review  https://lit-element.polymer-project.org/guide/properties#reflected-attributes
+  // to understand how to use reflected attributes with your property settings.
+  static get properties() {
+    return {
+      // ...super.properties,
+      day:           { type: Object },
+      selected:      { type: Boolean },
+      hovered:       { type: Boolean },
+      dateTo:        { type: String },
+      dateFrom:      { type: String },
+      month:         { type: String },
+      min:           { type: Number },
+      max:           { type: Number },
+      disabled:      { type: Boolean },
+      disabledDays:  { type: Array },
+      hoveredDate:   { type: String },
+      isCurrentDate: { type: Boolean },
+      locale:        { type: Object }
+    };
+  }
+
+  get locale() {
+    return this._locale ? this._locale : enUS;
+  }
+
+  set locale(value) {
+    const oldValue = this._locale;
+    this._locale = value;
+    this.requestUpdate('locale', oldValue);
+  }
+
+  static get styles() {
+    return [
+      // ...super.styles,
+      styleCss
+    ];
+  }
+
+  dateChanged(dateFrom, dateTo, hoveredDate, day) {
+    this.selected = false;
+    this.hovered = false;
+    const parsedDateFrom = parseInt(dateFrom, 10);
+    const parsedDateTo = parseInt(dateTo, 10);
+    if (day) {
+      if (getTime(startOfDay(parsedDateTo * 1000)) / 1000 === day.date || getTime(startOfDay(parsedDateFrom * 1000)) / 1000 === day.date) {
+        this.selected = true;
+      }
+      if (((hoveredDate === day.date || day.date < hoveredDate) && day.date > parsedDateFrom && !parsedDateTo && !Number.isNaN(parsedDateFrom) && parsedDateFrom !== undefined && !this.selected) || (day.date > parsedDateFrom && day.date < parsedDateTo)) {
+        this.hovered = true;
+      }
+    }
+  }
```

Comment:
This and all code is missing JSDoc comments?

Reply:
The functions here are just updating what was already in `wc-range-datepicker`. It's mostly a copy of their code with some slight tweaks to get the templates working the way we want.

I propose we clean up all the JSDoc for their functions when we cleanup all the actual JS code in a future release.

Reply:
@jason-capsule42 is there an issue created to address the work for adding comments to the functions?